src/core/ngx_connection.c

```diff
@@ -887,7 +887,16 @@ ngx_close_listening_sockets(ngx_cycle_t *cycle)
                      * for closed shared listening sockets unless
                      * the events was explicitly deleted
                      */
-
+#if (NGX_SSL)
+#if OPENSSL_VERSION_NUMBER >= 0x10100000L
```

Comment:
I suggest replacing it with "#if (NGX_SSL) && OPENSSL_VERSION_NUMBER >= 0x10100000L"

Reply:
fixed